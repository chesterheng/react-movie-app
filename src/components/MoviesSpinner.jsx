import React from 'react';
import { Spinner } from 'react-bootstrap';

const MoviesSpinner = () => {
  return (
    <Spinner animation="border" role="status">
      <span className="sr-only">Loading...</span>
    </Spinner>
  )
}

export default MoviesSpinner;
