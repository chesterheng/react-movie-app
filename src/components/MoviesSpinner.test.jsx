import React from "react";
import { render } from "@testing-library/react";
import { screen } from "@testing-library/dom";
import MoviesSpinner from "./MoviesSpinner";

it("should render MoviesSpinner component", () => {
  const { container } = render(<MoviesSpinner />);
  screen.debug();
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).toMatchInlineSnapshot(`
    <div
      class="spinner-border"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
  `);
});
