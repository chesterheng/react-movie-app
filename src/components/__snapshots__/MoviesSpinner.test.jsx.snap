// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MoviesSpinner component 1`] = `
<div
  class="spinner-border"
  role="status"
>
  <span
    class="sr-only"
  >
    Loading...
  </span>
</div>
`;
